Makefile

```diff
@@ -167,6 +167,11 @@ binary-e2e-serial:
 
 binary-e2e-all: binary-e2e-install binary-e2e-rte binary-e2e-sched binary-e2e-uninstall binary-e2e-serial
 
+runner-e2e-serial:
+	@grep -v 'source hack/common.sh' hack/run-test-e2e-serial.sh | \
```

Comment:
a little bit hackey, isn't it?

Reply:
yes, it is. Question is how we can reuse the same codebase for both use cases.